components/script_traits/lib.rs

```diff
@@ -473,9 +473,10 @@ pub enum CompositorEvent {
         Point2D<f32>,
         Option<UntrustedNodeAddress>,
         Option<Point2D<f32>>,
+        u16,
     ),
     /// The mouse was moved over a point (or was moved out of the recognizable region).
-    MouseMoveEvent(Option<Point2D<f32>>, Option<UntrustedNodeAddress>),
+    MouseMoveEvent(Option<Point2D<f32>>, Option<UntrustedNodeAddress>, u16),
```

Comment:
leave an inline comment on the `u16` saying it's a button mask of `MouseButton` values

(or mention this in the doc comment)